tests/pom.xml

```diff
@@ -129,10 +126,11 @@ Contributors:
 
 	</modules>
 
-
 	<properties>
 		<!-- jacoco (ECLEmma), code coverage -->
+		<maven-antrun-plugin.version>1.8</maven-antrun-plugin.version>
```

Comment:
remove, already defined in https://github.com/eclipse/n4js/blob/master/releng/org.eclipse.n4js.parent/pom.xml#L69

Reply:
done